language/cloud-client/v1/snippets.py

```diff
@@ -24,17 +24,20 @@
 import argparse
 import sys
 
-from google.cloud import language
-from google.cloud.language import enums
-from google.cloud.language import types
-import six
 
-
-# [START language_sentiment_text]
 def sentiment_text(text):
+    # [START language_sentiment_text]
+    import six
+    from google.cloud import language
+    from google.cloud.language import enums
+    from google.cloud.language import types
+
     """Detects sentiment in the text."""
```

Comment:
Please remove. It doesn't make sense to have a docstring if the user doesn't know it's a function.